camayoc/tests/qpc/cli/test_rhacs.py

```diff
@@ -0,0 +1,78 @@
+"""Tests for OpenShift sources.
```

Comment:
minor RHACS sources

Reply:
Thanks, fixed in aaf45569ff67d8d30e8faf6f3529901ab66a506f